keel-web/keel-web.gradle.kts

```diff
@@ -27,8 +27,8 @@ dependencies {
   implementation("com.netflix.spinnaker.fiat:fiat-api:${property("fiatVersion")}")
   implementation("com.netflix.spinnaker.fiat:fiat-core:${property("fiatVersion")}")
   implementation("net.logstash.logback:logstash-logback-encoder")
-  implementation("org.springdoc:springdoc-openapi-webmvc-core:1.2.32")
-  implementation("org.springdoc:springdoc-openapi-kotlin:1.2.32")
+  implementation("org.springdoc:springdoc-openapi-webmvc-core:1.4.1")
+  implementation("org.springdoc:springdoc-openapi-kotlin:1.4.1")
```

Comment:
Were these bumps required to get the tests passing, or just because you were you already touching the code?

Reply:
Just because I was already touching the code